### ISSUE TITLE
TypeError: 'NoneType' object is not subscriptable while generating images

### ISSUE THREAD

**Issue by onur222** [HanaokaYuzu/Gemini-API#18]
`  File "/gemini_webapi/client.py", line 370, in <listcomp>
    url=image[0][3][3],
        ~~~~~~~~^^^
TypeError: 'NoneType' object is not subscriptable`

I receive this error while running this code:

`    


    async def main():
        client = GeminiClient(proxies=None)
        # Create Layout

        response = await client.generate_content("Generate some pictures of cats")
        print(response)

    asyncio.run(main())
`

**Comment by HanaokaYuzu** [comment]
Cannot reproduce. Does this happen every time or randomly?

**Comment by onur222** [comment]
I happens every time. 
The code can produce images but links to those images are not returned.
I can see the images on https://gemini.google.com/app

I also can produce images on https://gemini.google.com/app with the same prompt.

**Comment by FuseFairy** [comment]
I have same issue.

**Comment by HanaokaYuzu** [comment]
Unfortunately this error never happens on my side and I'm not able to debug it without the response body you got. Guess that for some reason response structure can be different by account or region. 

Needs someone with basic python debugging skill to set a breakpoint [here](https://github.com/HanaokaYuzu/Gemini-API/blob/871faa1fc86d2fb0084b254b4424623e807042b0/src/gemini_webapi/client.py#L386) and find out the valid indexes (or simply print out and send the full `condidate` object to me).

https://github.com/HanaokaYuzu/Gemini-API/blob/871faa1fc86d2fb0084b254b4424623e807042b0/src/gemini_webapi/client.py#L386-L403

**Comment by HanaokaYuzu** [comment]
Fixed in #34
